user_data_model.md

```diff
@@ -0,0 +1,67 @@
+> NOTE: The following is implementation examples for discussion on future models for DAOs
+
+## User Data
+```rust
+pub struct ComposableUser {
+  pub attestations: LookupMap<AttestationUuid, Vec<Attestation>>,
+  pub attributions: LookupSet<Attribution>,
+  // User approved list of accounts they allow to submit attestations or attributions on this account.
+  // Default could be set with preferences, to either always allow new or always require approval
+  pub notaries: LookupSet<AccountId>,
+  // Allows user to control who & types of data allowed to be stored
+  pub preferences: UnorderedMap<Hash, Vec<u8>>,
```

Comment:
this might be just me, but these names are not very intuitive. What attestations should represent? What attributions should represent? Why do we need notaries.

Maybe I'm not understanding the purpose entirely, but everything seems over-engineered...